docs/model_pipeline.rst

```diff
@@ -93,9 +105,35 @@ Time series
    :file: ../oemof_b3/schema/timeseries.csv
 
 
-A few more conventions are important to know. Missing data is left empty. If a value applies to all
-regions, this is indicated by :attr:`ALL`. If it applies to the sum of regions, by :attr:`TOTAL`.
-There is no unit transformation within the model, i.e. the user needs to ensure the consistency of units.
+.. _conventions_resources_label_:
+
+Conventions
+^^^^^^^^^^^
+
+A few more conventions are important to know:
+
+* Missing data is left empty.
+* There is no unit transformation within the model, i.e. the user needs to ensure the consistency of units.
```

Comment:
All units are given in MW, MWh, Eur/MWh, ...?

Reply:
Did you had [this](https://github.com/rl-institut/oemof-B3/pull/280/commits/b9d2f64188fbe0347ce5d615c64e4691ce3cd1f3) in mind?

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/280/commits/90b5233c03bd8dd93800f8e143ec3bd6a25af8c4.